src/en/about/guide.md

```diff
@@ -1,57 +1,105 @@
+
+*You have the ability to view all public local posts that have been made by people on your server in the so-called “Local” tab.*
+
+If you aren’t here to stick mainly to one topic, you’ll likely want to pick a server that caters to general interests. Either way, there’s a helpful servers list on [join-lemmy.org](https://join-lemmy.org/instances).
+
+**Don’t panic!** You’ll be able to chatter to people on other servers, no matter which one you choose. Remember, this is like email and you can still email your mum on her ancient Yahoo account from your GMail, for example.
+
+## Knowing your server
+
+Take a moment before registering to check out the rules of your chosen server and make sure they are compatible with how you want to be posting.
+
+Servers usually define their rules in a simple way on the sidebar on the right of the site. From top to bottom, you will usually find the list of "Trending communities" followed by "Subscribed to communities", a brief description of the server, the rules, some stats and the username of the administrators.
+
+Admins are super friendly people who are usually running the server out of their own pocket so it’s good to get to know them like you would a landlord. Many accept donations to cover the running costs and if you’re able to contribute then it’s always appreciated.
+
+## I think I found my new home!
+
+Head to the Login page `(https://your.server/login)` of your server and choose your username and password in the sign up form. You don't need an email to register, but this is the only way to reset your password if you lose it.
+
+Next thing to do is upload your profile picture, give the settings page `(https://your.server/settings)` a good once-over (and do come back to it when you’ve been on Lemmy a week or so just to make any tweaks that might help your experience) and get ready to introduce yourself.
+
+*Some interesting settings worth checking are: your default feed sorting filter bewteen subscribed, local or all and your default sorting type.*
+
+While you can simply enjoy reading Lemmy, the real fun begins as you dive in and start participating yourself by posting, voting and commenting.
+
+To share a post, click the “Create Post” button on the top side of the screen. You’ll be taken to a form where you can enter an URL of the link or upload your image and input a title and body for the post. You will also need to enter the communitie where you’d like your post to appear. Before doing so, it’s a good idea to head to that communitie’s page and read the posting guidelines that appear on the right side of the screen. This will ensure you are abiding by the particular ethos of the communitie. For example, on the `!privacy` communitie, you are not allowed to promote proprietary software.
```

Comment:
"communitie" should be community (maybe run a spell check)

Reply:
and you should put !privacy@lemmy.ml (maybe with a direct link to the community for easy viewing)